catkit/hardware/boston/DmCommand.py

```diff
@@ -262,26 +261,18 @@ def convert_m_to_volts(data, dm_num, meter_to_volt_map=None):
 
 
 def convert_dm_command_to_image(dm_command):
-    # Flatten the mask using index952
-    mask = catkit.util.get_dm_mask()
-    index952 = np.flatnonzero(mask)
+    mask = catkit.util.get_dm_mask().astype('bool')
+    number_of_actuators = np.count_nonzero(mask)
 
-    number_of_actuators_per_dimension = CONFIG_INI.getint('boston_kilo952', 'dm_length_actuators')
-    number_of_actuators = CONFIG_INI.getint("boston_kilo952", "number_of_actuators")
-    image = np.zeros((number_of_actuators_per_dimension, number_of_actuators_per_dimension))
-    image[np.unravel_index(index952, image.shape)] = dm_command[:number_of_actuators]
+    image = np.zeros(mask.shape)
+    image[mask] = dm_command[:number_of_actuators]
```

Comment:
👍  Good reduction! It took me way too long to even figure out what the old code was doing.